config/routes.rb

```diff
@@ -122,9 +122,7 @@
       end
     end
 
-    constraints(:hostgroup => /[^\/]+/) do
-      get 'unattended/template/:id/:hostgroup', :to => "unattended#hostgroup_template"
-    end
+    get 'unattended/template/:id/*hostgroup', :to => "unattended#hostgroup_template", hostgroup: /.+/
```

Comment:
Using route globbing, for supporting nested hostgroup. the constraint had changed to support in dots in the hostgroup / parents name.